bmSettings.lua

```diff
@@ -1,9 +1,13 @@
-local mq            = require('mq')
-local btnUtils      = require('lib.buttonUtils')
-
-local settings_base = mq.configDir .. '/ButtonMaster'
-local settings_path = settings_base .. '.lua '
+local mq                         = require('mq')
+local btnUtils                   = require('lib.buttonUtils')
+local PackageMan                 = require('mq.PackageMan')

```

Comment:
This will break only init.lua can invoke packageman - you can use it here but you ALSO need to add it to init.lua

Reply:
This only works if it is a no-op. So there is no reason to have it at all. Just do a normal require.